bouncer/blacklist/views.py

```diff
@@ -9,8 +10,34 @@
 )
 
 
-class IPRequestView(View):
+class RequestView(View):
+    template = loader.get_template("blacklist/request.html")
+
     def get(self, request, *args, **kwargs):
-        if is_ip_blacklisted(request.GET["ip"]):
-            return HttpResponse("YES")
-        return HttpResponse("NO")
+        query = dict(self.request.GET.lists())
+        context = {"results": []}
```

Comment:
It would be best to use `namedtuple`s here, you can read more about named tuples here: https://docs.python.org/3.6/library/collections.html#collections.namedtuple